bundles/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/handler/PLCBridgeHandler.java

```diff
@@ -93,48 +105,52 @@ public void run() {
 
         @Override
         public void run() {
-            PLCLogoClient localClient = client;
-            Map<?, Layout> memory = LOGO_MEMORY_BLOCK.get(getLogoFamily());
-            Layout layout = (memory != null) ? memory.get(MEMORY_SIZE) : null;
-            if ((layout != null) && (localClient != null)) {
+            final var client = PLCBridgeHandler.this.client;
+            final var memory = LOGO_MEMORY_BLOCK.get(getLogoFamily());
+            final var layout = (memory != null) ? memory.get(MEMORY_SIZE) : null;
+            if ((layout != null) && (client != null)) {
```

Comment:
Nitpick: no need to have the extra sets
```suggestion
            if (layout != null && client != null) {
```

Reply:
Cross-Checked the source style: There is no rule for this available. Therefore, i would say, we speak about personal preferences...

Reply:
We don’t have explicit rules for every situation, but I’ve seen fellow maintainers ask to remove extra parentheses in similar cases. They don’t add value and can make the code harder to read. Since Java already evaluates conditions with short-circuiting (&&), the simpler form is sufficient.